src/client/sandbox/console.js

```diff
@@ -0,0 +1,27 @@
+import SandboxBase from './base';
+
+export default class ConsoleSandbox extends SandboxBase {
+    constructor () {
+        super();
+
+        this.CONSOLE_METH_CALLED = 'hammerhead|console|console-meth-called-event';
+    }
+
+    attach (window) {
+        super.attach(window);
+
+        const sandbox = this;
+
+        const proxyConsoleMeth = meth => {
```

Comment:
Make this function as private member of ConsoleSandbox, I think it is more redable.